src/scripts/main.js

```diff
@@ -1,3 +1,5 @@
 'use strict';
-
-// write code here
+const spider = document.querySelector('.spider');
+spider.style.top = '50%'
```

Comment:
There is a missing semicolon at the end of this line. In JavaScript, it's a good practice to terminate statements with a semicolon to avoid potential issues with automatic semicolon insertion.